_vector-search/ai-search/hybrid-search/inner-hits.md

```diff
@@ -0,0 +1,648 @@
+
+The response includes an `_explanation` object with detailed scoring information. The nested `details` array provides the relevant information about the score mode used, the number of child documents contributing to the parent document's score, and how the scores were normalized and combined:
+
+```json
+{
+  ...
+  "_explanation": {
+    "value": 1.0,
+    "description": "arithmetic_mean combination of:",
+    "details": [
+      {
+        "value": 1.0,
+        "description": "min_max normalization of:",
+        "details": [
+          {
+            "value": 0.4458314776420593,
+            "description": "combined score of:",
+            "details": [
+              {
+                "value": 0.4394061,
+                "description": "Score based on 1 child docs in range from 0 to 6, using score mode Avg",
+                "details": [
+                  {
+                    "value": 0.4394061,
+                    "description": "weight(user.name:john in 0) [PerFieldSimilarity], result of:"
+                    // Additional details omitted for brevity
+                  }
+                ]
+              },
+              {
+                "value": 0.44583148,
+                "description": "Score based on 1 child docs in range from 0 to 6, using score mode Avg",
+                "details": [
+                  {
+                    "value": 0.44583148,
+                    "description": "weight(location.city:udaipur in 5) [PerFieldSimilarity], result of:"
+                    // Additional details omitted for brevity
+                  }
+                ]
+              }
+            ]
+          }
+        ]
+      }
+    ]
+  }
+}
+...
+```
+
+## Sorting with inner hits
+
+To apply sorting, add a `sort` subclause in the `inner_hits` clause. For example, to sort by `user.age`, specify this sort condition in the `inner_hits` clause:
+
+```json
+GET /my-nlp-index/_search?search_pipeline=nlp-search-pipeline
+{
+  "query": {
+    "hybrid": {
+      "queries": [
+        {
+          "nested": {
+            "path": "user",
+            "query": {
+              "match": {
+                "user.name": "John"
+              }
+            },
+            "score_mode": "sum",
+            "inner_hits": {
+              "sort": [
+                {
+                  "user.age": {
+                    "order": "desc"
+                  }
+                }
+              ]
+            }
+          }
+        },
+        {
+          "nested": {
+            "path": "location",
+            "query": {
+              "match": {
+                "location.city": "Udaipur"
+              }
+            },
+            "inner_hits": {}
+          }
+        }
+      ]
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+In the response, the `user` inner hits are sorted by age in descending order rather than by relevance, which is why the `_score` field shows as `null` (scores are not calculated when custom sorting is applied):
+
```

Comment:
Line 420: I chose "displays" instead of "shows", but would something like "returns" be more accurate?